did the javascript part

Pratchaveea opened the pull request:
did the javascript part

Pratchaveea committed:
did the javascript part

NikoMastro commented:
I'm on the resolution

NikoMastro commented on app/views/layouts/application.html.erb:
```diff
@@ -6,6 +6,7 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.1.2/css/all.css">
```
We don't need the line here, we already call FA in another file :)

NikoMastro committed:
Update application.html.erb

No need to import the font awesome, it's called in main index

NikoMastro committed:
Delete app/views/products/show.html.erb

the code of this js on product has been copy/pasted.
Working on issue and will call it in external file
